dr_backup/dr_backup.py

```diff
@@ -0,0 +1,304 @@
+import boto3
+import time
+import os
+
+
+def delete_snapshot(rds_client, id, cluster=True):
+    snapshot_type = "manual"
+    try:
+        if cluster:
+            rds_client.describe_db_cluster_snapshots(
+                DBClusterSnapshotIdentifier=id,
+                SnapshotType=snapshot_type
+            )
+        else:
+            rds_client.describe_db_snapshots(
+                DBSnapshotIdentifier=id,
+                SnapshotType=snapshot_type
+            )
+        print(f"Snapshot {id} exists, deleting...")
+
+        if cluster:
+            rds_client.delete_db_cluster_snapshot(
+                DBClusterSnapshotIdentifier=id
+            )
+        else:
+            rds_client.delete_db_snapshot(
+                DBSnapshotIdentifier=id
+            )
```

Comment:
Could be my non-existent python skills but could these blocks be merged with the ones above if all we're checking `cluster = True`?

Reply:
Nah just didn't want to do the print twice but yeah saves a couple of lines of code so done it